pubspec.yaml

```diff
@@ -1,22 +1,23 @@
 name: google_api_headers
 description: A Flutter plugin for getting the headers required for calling Google APIs with an app restricted key.
-version: 0.1.1+2
+version: 0.2.0-nullsafety.0
```

Comment:
Don't worry about updating the version, I'll update it before I publish the package